kitsune/coolsearch/static/js/Result.jsx

```diff
@@ -0,0 +1,10 @@
+export default class Result extends React.Component {
```

Comment:
Calling this file Result might be problematic. Although it seems like everything is namespaced with the Django app, JS files all get put into the same directory. I didn't figure out how to deal with this with the community hub React stuff. You may want to call this SearchResult, or maybe namespace all these files as `kitsune/coolsearch/static/js/coolsearch/Result.jsx`.